lib/plugins/aws/deploy/lib/uploadArtifacts.js

```diff
@@ -140,7 +139,7 @@ module.exports = {
         .filter(Boolean)
     );
 
-    return BbPromise.map(
+    return promiseMap(
```

Comment:
Let's use `Promise.all`, and we can control concurrency as:

```javascript
const limit = require('ext/promise/limit).bind(Promise)

Promise.all(limit(MAX_CONCURRENT_ARTIFACTS_UPLOADS, async () => {...})
```

`ext` is currently not listed as our dependency, but it's used by our dependencies, so that technically doesn't add to our dependencies baggage, and it's free to add it.

Reply:
@medikoo Currently, latest version of [ext@1.4.0](https://www.npmjs.com/package/ext/v/1.4.0) published on NPM does not include `promise/limit` module. It is also hard to follow the versioning, as the releases on GitHub use two different versioning schemes. Could you please help with providing proper version to get promise/limit in place?

Reply:
@remi00 indeed! Sorry I was completely sure it's published.

It's published now with v1.5.0